src/main/webapp/usermap.html

```diff
@@ -55,8 +55,13 @@
             <h1 id="caption">Who is using TEAMMATES?</h1>
             <div id="contentHolder">
                 <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE"></div>
-                <p id="totalCount"></p>
-                <p>Last updated: 30 March 2016</p>
+                    <p id="lastUpdate" class="lastUpdate">Last updated: 14 March 2016</p>
+                 	<h2 class="subcaption align-center">
+                      <span id="totalUserCount" class="totalCount"></span> 
+                      institutions from 
+                      <span id="totalCountryCount" class="totalCount"></span> 
+                      countries
+                   </h2>
```

Comment:
Yes. Don't forget, this issue is not about changing the last updated date. In a group project, you shouldn't always expect others to steer clear from the parts of code that you're touching. If others updated similar parts of the code, you should be able to adapt without drifting away from what you're supposed to do.